### DISCUSSION TITLE
How to reset a layout to default

### DISCUSSION THREAD

**Discussion by BrandonBordeaux** [ianyh/Amethyst#1507] [Q&A] [answered]
I don't know how I did this, as I was tired and used an unknown key combination, but I've altered the Tall layout on my main screen so that it arranges windows in rows. On other screens, the Tall layout arranges as expected. How do I reset the Tall layout on my first screen so it behaves per the default?

**Comment by ianyh** [comment, marked as answer]
You probably increased the main pane count too far. You can decrease the main pane count back to 0. You can also turn off restoring layouts on launch and relaunch.

**Reply by BrandonBordeaux** [reply]
Thank you! Reducing the main pane count to 0 didn't resolve the issue, but turning off `Restore layouts on launch` and relaunching did.
